Src/PChecker/CheckerCore/SystematicTesting/Strategies/Feedback/Generator/RandomChoices.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using PChecker.Exceptions;
+
+namespace PChecker.Generator.Object;
+
+public class RandomChoices<T>
+where T: IConvertible
+{
+    internal readonly System.Random Random;
+    public int Pos;
```

Comment:
There seems to be bug here, `pos` is never initialized to 0?

Reply:
This is not necessary. Pos is default to 0 during initialization.

Reply:
How and where?

Reply:
C# makes sure Pos is initialized to 0. 

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/default-values and https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1805